adapter/pyproject.toml

```diff
@@ -8,15 +8,20 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.7.1, <3.11"
+python = ">=3.7.2, <3.11"
 dbt-core = ">=1.3.0b1, <1.4"
-fal = ">=0.6.0"
 isolate = ">=0.3.0"
+posthog = "^1.4.5"
+pandas = "^1.3.4"
+"backports.functools_lru_cache" = "^1.6.4"
```

Comment:
wow, what's this?

Reply:
What we use for cache function in Python 3.7